.vib/airflow/goss/airflow.yaml

```diff
@@ -0,0 +1,20 @@
+command:
+  check-scheduler:
+    timeout: 30000
+    exec: airflow users list
+    exit-status: 0
+    stdout:
+      - No data found
+  check-so-files:
+    # Check that all .so files in site-packages are executable
+    exec: find /opt/bitnami/airflow/venv/lib/python3.9/site-packages/ -type f -name '*.so.*' ! -executable | wc -l
+    exit-status: 0
+    stdout:
+      - 0
+file:
+  /opt/bitnami/airflow/venv/lib/python3.9/site-packages/airflow/www/node_modules:
+    exists: false
+  /opt/bitnami/airflow/venv/lib/python3.9/site-packages/airflow/config_templates/default_airflow.cfg:
+    exists: true
+    contains:
+      - Typically, this is a simple statement like "SELECT 1"
```

Comment:
This is no longer relevant, the issue this refers to was solved in upstream:

https://github.com/apache/airflow/blob/main/airflow/config_templates/default_airflow.cfg#L307

In fact, you should create an internal task to solve it

Reply:
I will create a task